vendors/st/boards/stm32l475_discovery/aws_demos/config_files/iot_pkcs11_config.h

```diff
@@ -83,6 +83,15 @@
  */
 #define pkcs11configJITP_CODEVERIFY_ROOT_CERT_SUPPORTED    0
 
+/**
+ * @brief Disable ECDSA crypto algorithm (i.e. mechanism) support in this
+ * PKCS #11 module.
+ *
+ * Set this to 1 when the host doesn't support ECDSA for network authentication
+ * (e.g. in the TLS protocol implementation).
+ */
+#define pkcs11configSUPPRESS_ECDSA_MECHANISM               1
```

Comment:
For TI, below, you've got the equivalent change for Test as well as Demo. But for STM32, you've only got it for Demo. Is that intentional?

Reply:
ST already had this configuration, that is why this issue did not exist on the test project of ST.